src/generate-api-types.test.ts

```diff
@@ -171,6 +172,9 @@ export const Schema: OneSchema<Endpoints> = {
 import type { OneSchema } from "@lifeomic/one-schema";
 
 export type Endpoints = {
+  /**
+   * Fetches an array of all fruits.
+   */
```

Comment:
This is the key change :point_up:: the route description is included on each route in `Endpoints`, when provided.

Reply:
This type still gets generated for clients (even in the case of a non-YAML flow). Will be great to have client-side.

Reply:
Oh excellent, good to hear.